src/tools/rbd/action/Status.cc

```diff
@@ -146,8 +156,25 @@ static int do_show_status(librados::IoCtx& io_ctx, const std::string &image_name
       f->dump_string("state_description", migration_status.state_description);
       f->close_section(); // migration
     }
-    if (!image_cache_str.empty()) {
-      f->dump_string("image_cache_state", image_cache_str);
+    if (!persistent_cache_str.empty()) {
+      auto& p = persistent_cache_state;   // shot name
+      f->open_object_section("persistent_cache");
```

Comment:
I still don't know how to test this code block. Just wrote this part of the code according to your example, and there is no timestamp and hit rate calculation

Reply:
found the method, output:
```
root@cephs:~# rbd status test/image1 --format json
{"watchers":[{"address":"10.239.241.64:0/1732150256","client":5066,"cookie":94783236491520}],"persistent_cache":{"stats_timestamp":1649211223,"mode":"rwl","present":true,"empty":false,"clean":false,"pwl_host":"cephs","pwl_path":"/mnt/pmem/cache//rbd-pwl.test.104666de1224.pool","pwl_size":10737418240,"pwl_bytes_allocated":314138624,"pwl_bytes_available":7202054144,"pwl_bytes_dirty":442368,"pwl_requests_read":0,"pwl_requests_read_hit":0,"pwl_bytes_read":0,"pwl_bytes_read_hit":0}}
root@cephs:~# rbd status test/image1 --format xml
<status><watchers><watcher><address>10.239.241.64:0/1732150256</address><client>5066</client><cookie>94783236491520</cookie></watcher></watchers><persistent_cache><stats_timestamp>1649211228</stats_timestamp><mode>rwl</mode><present>true</present><empty>false</empty><clean>false</clean><pwl_host>cephs</pwl_host><pwl_path>/mnt/pmem/cache//rbd-pwl.test.104666de1224.pool</pwl_path><pwl_size>10737418240</pwl_size><pwl_bytes_allocated>419880960</pwl_bytes_allocated><pwl_bytes_available>7096311808</pwl_bytes_available><pwl_bytes_dirty>917504</pwl_bytes_dirty><pwl_requests_read>0</pwl_requests_read><pwl_requests_read_hit>0</pwl_requests_read_hit><pwl_bytes_read>0</pwl_bytes_read><pwl_bytes_read_hit>0</pwl_bytes_read_hit></persistent_cache></status>
```